src/payments/components/EventPayment/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, useEffect, useState } from 'react';
+
+import Spinner from 'common/components/Spinner';
+import HttpError from 'core/components/errors/HttpError';
+
+import { getAttendanceEvent } from 'events/api/events';
+import { IUserAttendee } from 'events/models/Attendee';
+import { getEventUserAttendees } from 'payments/api/paymentRelation';
+import { IPaymentPrice } from 'payments/models/Payment';
+import { Payment } from '../Payment';
+import style from '../Payment/payment.less';
+
+interface IProps {
+  eventId: number;
+}
+
+export const EventPayment: FC<IProps> = ({ eventId }) => {
+  const [userAttendees, setUserAttendees] = useState<IUserAttendee[]>();
```

Comment:
I almost feel like it's better as-is, because we show a spinner when there are no user attendees. If we add an empty array we would need to check if that array is non-empty. I guess since you're always attending this event yourself if you reach the page that's going to hold true, but it seems even more non-trivial to me :stuck_out_tongue:

Reply:
Sounds reasonable :smile: